src/main/java/uk/gov/pay/ledger/transactionsummary/dao/TransactionSummaryDao.java

```diff
@@ -152,4 +164,35 @@ public PerformanceReportEntity performanceReportForPaymentTransactions(Performan
             return query.map(new PerformanceReportEntityMapper()).one();
         });
     }
+
+    public TransactionsStatisticsResult getTransactionSummaryStatistics(TransactionSummaryStatisticQuery transactionSummaryStatisticQuery, TransactionType transactionType) {
+        return jdbi.withHandle(handle -> {
+            String template = createSearchTemplate(transactionSummaryStatisticQuery.getFilterTemplates(), TRANSACTION_SUMMARY_STATISTICS);
```

Comment:
This could be outside of the return statement

Reply:
Done.